version.go

```diff
@@ -3,10 +3,11 @@ package gochinadns
 import "fmt"
 
 const (
-	name    = "GoChinaDNS"
-	version = "v1.0"
+	name = "GoChinaDNS"
 )
 
+var version = "v1.0"
```

Comment:
```suggestion
var version = "v1.6"
```

It would be nice to bump to a new version 😄

Reply:
oh I see, it's just a placeholder now

Reply:
Yes it's just a placeholder, it's set during build with ldflags -X 😀 You could also change it to  `var version string` that way it will be empty by default.

Feel free to squash the commits. I made a mistake with updating the dependencies so there are 3 commits for it, you can squash all of them.